services-camel/src/test/java/edu/unc/lib/dl/services/camel/fulltext/FulltextProcessorTest.java

```diff
@@ -80,69 +74,28 @@
     @Before
     public void init() throws Exception {
         initMocks(this);
-        processor = new FulltextProcessor(repoObjLoader, slug, fileName, maxRetries, retryDelay);
-        file = File.createTempFile(fileName, "txt");
-        file.deleteOnExit();
-        when(exchange.getIn()).thenReturn(message);
 
-        when(repoObjLoader.getBinaryObject(any(PID.class))).thenReturn(binary);
+        processor = new FulltextProcessor(tmpDir.newFolder().getAbsolutePath());
+        file = tmpDir.newFile(fileName + ".txt");
```

Comment:
this makes for file `full_text.txt.txt`, so you can leave off the concatenation here.